superset-frontend/src/views/CRUD/data/dataset/AddDataset/DatasetPanel/DatasetPanel.tsx

```diff
@@ -54,8 +56,10 @@ const MARGIN_MULTIPLIER = 3;
 
 const StyledHeader = styled.div<StyledHeaderProps>`
   position: ${(props: StyledHeaderProps) => props.position};
-  margin: ${({ theme }) => theme.gridUnit * MARGIN_MULTIPLIER}px;
-  margin-top: ${({ theme }) => theme.gridUnit * (MARGIN_MULTIPLIER + 1)}px;
+  margin: ${({ theme }) => theme.gridUnit * (MARGIN_MULTIPLIER + 1)}px
+    ${({ theme }) => theme.gridUnit * MARGIN_MULTIPLIER}px
+    ${({ theme }) => theme.gridUnit * MARGIN_MULTIPLIER}px
+    ${({ theme }) => theme.gridUnit * (MARGIN_MULTIPLIER + 3)}px;
```

Comment:
I believe you can make this just one function for all margin sides

Reply:
These docs address how you can get props once and not have to make multiple inline functions: https://github.com/apache/superset/wiki/Emotion-Styling-Guidelines-and-Best-Practices#no-need-to-use-theme-the-hard-way

Reply:
Oh yeah, that is a lot cleaner! I cleaned up this and some other spots that could be cleaned in the same way in [`this commit`](https://github.com/apache/superset/pull/22136/commits/86549660bd4d6ac53e0b7723ab3e9370e37d2a28).

Reply:
Looks good!